static/src/scss/resources/_categories.scss

```diff
@@ -159,7 +186,7 @@
             height: 0;
             border-left: 0.675rem solid transparent;
             border-right: 0.675rem solid transparent;
-            border-top: 0.675rem solid #000000;
+            border-top: 0.675rem solid darken(#50e3c2, 20%);
```

Comment:
Colours are stored in variables, in this instance it would be `$color-accent`

Reply:
$color-accent works everywhere but within darken, has an error saying it requires an actual colour. $color-accent-fixed works though, is that ok to use?

Reply:
Yeah that works!